src/MigrationBuilder.php

```diff
@@ -47,8 +51,8 @@ public function getDb(): ConnectionInterface
     public function execute(string $sql, array $params = []): void
     {
         $sqlOutput = $sql;
-        if ($this->maxSqlOutputLength > 0) {
-            $sqlOutput = StringHelper::truncateEnd($sql, $this->maxSqlOutputLength, '[... hidden]');
+        if (0 < $this->maxSqlOutputLength && $this->maxSqlOutputLength < strlen($sql)) {
+            $sqlOutput = substr($sql, 0, $this->maxSqlOutputLength - 12) . '[... hidden]';
```

Comment:
```suggestion
            $sqlOutput = substr($sql, 0, $this->maxSqlOutputLength) . ' [... hidden]';
```

I think `maxSqlOutputLength` should specify SQL output length only, but not full console output string. What do you think?

Reply:
Yes, agree. And it sovles problem with `$this->maxSqlOutputLength` less than `strlen('[... hidden]')`.